### DISCUSSION TITLE
[CRITICAL] [nextdns] Certificate hash not found

### DISCUSSION THREAD

**Discussion by MojtabaFaal** [DNSCrypt/dnscrypt-proxy#1808] [Q&A] [answered]
Hi
What is this CRITICAL error?

`[2021-08-08 14:28:44] [CRITICAL] [nextdns] Certificate hash [cc1060d39c8329b62b6fbc7d0d6df9309869b981e7e6392d5cd8fa408f4d80e6] not found`

I receive this CRITICAL error yesterday and today
What should I do ?

**Comment by jedisct1** [comment, marked as answer]
Looks like they just switched to a new certificate authority.

Delete the `public-resolvers.md` file and restart the server. It should load the new version.

**Reply by MojtabaFaal** [reply]
Hi
A suggestion
Isn't it better for the program to try to get the new certificate every time it encounters this error, if the new certificate was valid, then the program deletes the `public-resolvers.md` file and receives the new version of the file?
